src/core/surrogate_test.jl

```diff
@@ -1,17 +1,20 @@
 using Random: AbstractRNG
 import StatsAPI: HypothesisTest, pvalue
 export SurrogateTest, pvalue
+using Threads
```

Comment:
The tests fails because `Threads` is not in the dependency list for the package. Using `Base.Threads` instead should solve it, I think.